containers/RepositoryPullRequestScreen/RepositoryPullRequestScreen.js

```diff
@@ -101,6 +115,36 @@ class RepositoryPullRequestScreen extends PureComponent<Props> {
                     </View>
                 </View>
                 <UIText style={styles.body}>{pullRequest.body}</UIText>
+                <ImageButton
+                  style={styles.button}
+                  onPress={() => this.props.addModal(
```

Comment:
Such kind of function passing is the anti-pattern.
[Related reading](https://medium.com/@esamatti/react-js-pure-render-performance-anti-pattern-fb88c101332f#9bed)